CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ to [Semantic Versioning](http://semver.org/).
   new releases of the site.
 - The dev version modal now links to the latest release
   of the site if the current git revision is unavailable.
+- New homepage design
```

Comment:
Sorry, this is something I'm kind of a curmudgeon about--please use complete sentences in the changelog :grin: